### DISCUSSION TITLE
Search results filtered by unique values

### DISCUSSION THREAD

**Discussion by smiliehub** [milvus-io/milvus#4690] [Q&A and General discussion]
Hi, 
So I was experimenting with milvus by adding other parameters/fields in the collection. So instead of just the embedding, I added something like a year value. Now I was curious if its possible to do a query search wherein the results would be unique by the year value similar for example in the top 10 results I get the most similar embedding each for year 2000-2009. Is this possible?

**Comment by yhmo** [comment]
Which version of milvus? As we have announce to stop maintain for 0.11.x(experimental). The recommended version is 0.10.x. The 0.10.x doesn't support multi-field, so I think you can use "partition" to distribute your embeddings.
For example, I create a collection, then create a partition for each year, give a name for each partition, "Y_2000", "Y_2001", ... "Y_2009". Then I perform search, specify the year/partition which I want to query:
`milvus.search(collection_name, top_k, query_records, partition_tags=[“Y_2001”, "Y_2008"])`
